### DISCUSSION TITLE
bmarker加右键rightclick不起效果  求解！！

### DISCUSSION THREAD

**Discussion by 3246046194** [yue1123/vue3-baidu-map-gl#77] [Q&A] [answered]
<BMarker
								v-if="stageRingGraph"
								v-for="(cross, index) in crossesStore.crosses"
								:visible="
									currentModeActive === '0' ||
									currentModeActive === cross.controlMode
								"
								:key="cross.CrossNumber"
								:position="{
									lng: cross.Lng,
									lat: cross.Lat
								}"
								:zIndex="99"
								@rightclick="handleRightClickMarker(1)"
								:icon="getmarkerIcon(cross)"
								ref="BMarkerref"
								:enable-dragging="false"
								@initd="(params) => handleMarkerInited(cross, params)"
								@mouseover="handleMouseoverMarker(cross)"
								@mouseout="handleMouseoutMarker(cross)"
								@click="handleClickMarker(cross)"
							/>
求解！！

**Comment by 3246046194** [comment]
偶然会触发不知道是哪里的问题，此方法加载map上可以正常触发！

**Comment by yue1123** [comment, marked as answer]
这貌似是一个小bug，请提一个 issue， 我会尽快解决
